pages/ksphere/konvoy/1.5/tutorials/disable-audit-logs/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: layout.pug
+navigationTitle: Disable Audit Log Collection
+title: Disable Audit Log Collection
+menuWeight: 11
+excerpt: Audit Logs are collected via the FluentBit Addon and enabled for collection by default. In some cases this may not be necessary and turning off audit-log-collection would be ideal. Here we show the procedure to disable Audit Log Collection.
+enterprise: false
+---
+
+### Disable Audit Log Collection
+
+Audit Logs are collected via the FluentBit Addon and enabled for collection by default. In some cases this may not be necessary and turning off audit-log-collection would be ideal. Here we show the procedure to disable Audit Log Collection.
```

Comment:
```suggestion
Audit Logs are collected using the FluentBit Addon and enabled for collection by default. In some cases this may not be necessary and turning off audit-log-collection would be ideal. This procedure shows how to disable Audit Log Collection.
```